configuration/ignored-service-accounts.md

```diff
@@ -0,0 +1,26 @@
+---
+description: This article explains what Service Accounts are and how to Ignore Service Account activities within SysKit Point.
+---
+
+# Ignored Service Accounts
```

Comment:
odmah iza naslova bi prvo stavio što su to service accounti - rečenica od ispod
**Service accounts are user accounts created specifically to fulfill automated, expected changes to the server.**
Umjesto server staviti environment.

dodao bih još i primjer:
For example, if you are using an automated backup solution, chances are there is a service account performing backups in defined intervals. 

For example, if you are using an automated backup solution, chances are there is a service account performing backups in defined intervals. 
Unified audit logs are created as a result of service account activity, the same as for activities of regular users.
i dalje onda povezano sa ovim ispod
The **Ignored Service Accounts** addition to the **General Settings** of Syskit Point allows you to avoid tracking service accounts activities collected from Unified Audit Logs, meaning: